deployments/pulse3d/manifests/base/pulse3d-dep.yaml

```diff
@@ -19,7 +19,6 @@ spec:
     spec:
       containers:
       - name: pulse3d
-        image: 077346344852.dkr.ecr.us-east-2.amazonaws.com/pulse3d_api:0.5.0
```

Comment:
we probably need to add this to the test overlay kustomization since there is no default now